concrete/controllers/single_page/account/edit_profile.php

```diff
@@ -136,9 +115,6 @@ public function save()
             $controller = $uak->getController();
             $validator = $controller->getValidator();
             $response = $validator->validateSaveValueRequest($controller, $this->request, $uak->isAttributeKeyRequiredOnProfile());
-            /**
-             * @var $response ResponseInterface
```

Comment:
Why did you remove this? I think this should stay unless `validateSaveValueRequest` gets a proper docblock

Reply:
Because we had a `use` statement just for this phpdoc.

Reply:
So?